### ISSUE TITLE
Remove operatorkit dependency from silence-operator

### ISSUE THREAD

**Issue by JosephSalisbury** [giantswarm/roadmap#3729]

**Comment by JosephSalisbury** [comment]
fyi @Rotfuks / @QuentinBisson, tl;dr - we want to get rid of operatorkit, silence-operator still uses it

probably the most straightforward plan here would be to move to flux for the silence syncing, and then rewrite the remaining behaviour in a new kubebuilder operator, but happy to discuss

i'd like to get this done, but there's no real urgency on it, it's more on the technical debt side

**Comment by QuentinBisson** [comment]
I agree on your plan but I'm pretty sure this will not be prioritized before 6-9 months and we work on our silence epic

**Comment by Rotfuks** [comment]
Are you happy with "next year" for this to be done, @JosephSalisbury ?

**Comment by JosephSalisbury** [comment]
works for me, as long as you have a plan for it <3

**Comment by QuentinBisson** [comment]
Work is ongoing;
https://github.com/giantswarm/silence-operator/pull/532
https://github.com/giantswarm/silence-operator/pull/534
https://github.com/giantswarm/silence-operator/pull/535
https://github.com/giantswarm/silence-operator/pull/536

**Comment by QuentinBisson** [comment]
Last one I think https://github.com/giantswarm/silence-operator/pull/539/
